gnc/sub8_perception/srv/TorpBoardPoseRequest.srv

```diff
@@ -1,6 +1,6 @@
 geometry_msgs/PoseStamped pose_stamped
-int32[12] l_proj_mat
-int32[12] r_proj_mat
+float32[12] l_proj_mat
+float32[12] r_proj_mat
```

Comment:
lol I don't know how I didn't catch it when you made it so explicit last time...